<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editorClass</class>
 <widget class="QMainWindow" name="editorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDI ~ Editor de imagens</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scroll_img1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>510</width>
       <height>510</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_img1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>512</width>
        <height>512</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scroll_img2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>510</width>
       <height>510</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_img2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>512</width>
        <height>512</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ü°Ñ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Arquivo">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="action_Abrir"/>
   </widget>
   <widget class="QMenu" name="menu_Operacoes">
    <property name="title">
     <string>&amp;Opera√ß√µes</string>
    </property>
    <addaction name="action_paraCinza"/>
    <addaction name="actionDividir_RGB"/>
    <addaction name="actionInverte_cinza"/>
    <addaction name="actionInverte_colorida"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionGuia_b_sico"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menu_Arquivo"/>
   <addaction name="menu_Operacoes"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Abrir">
   <property name="text">
    <string>&amp;Abrir</string>
   </property>
  </action>
  <action name="action_paraCinza">
   <property name="text">
    <string>&amp;Converter para cinza</string>
   </property>
  </action>
  <action name="actionGuia_b_sico">
   <property name="text">
    <string>Guia b√°sico</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionDividir_RGB">
   <property name="text">
    <string>Dividir RGB</string>
   </property>
  </action>
  <action name="actionInverte_cinza">
   <property name="text">
    <string>Inverter imagem cinza</string>
   </property>
  </action>
  <action name="actionInverte_colorida">
   <property name="text">
    <string>Inverter imagem colorida</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections/>
</ui>
